taxi/forms.py

```diff
@@ -0,0 +1,53 @@
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+from django import forms
+from django.core.exceptions import ValidationError
+
+from taxi.models import Driver, Car
+
+
+def validate_license_number(license_number):
+    if len(license_number) != 8:
+        raise ValidationError(
+            f"Ensure this value has at most 8 characters (it has "
+            f"{len(license_number)})."
+        )
+    if not license_number[3:8].isdigit():
+        raise ValidationError("4-8 must be numbers!")
+    if not license_number[:3].isalpha() or not license_number[:3].isupper():
+        raise ValidationError("1-3 must be uppercase chars!")
+    return license_number
+
+
+class DriverCreationForm(UserCreationForm):
+    license_number = forms.CharField(
+        required=True,
+        validators=[validate_license_number]
+    )
+
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + ("license_number",)
+
+
+class CarForm(forms.ModelForm):
+    drivers = forms.ModelMultipleChoiceField(
+        queryset=get_user_model().objects.all(),
```

Comment:
The queryset for the `drivers` field should be limited to `Driver` instances, not all user instances. Consider using `Driver.objects.all()` instead of `get_user_model().objects.all()` to ensure only drivers are selectable.